app/controllers/clinics_controller.rb

```diff
@@ -23,9 +23,9 @@ def create
   end
 
   def new
-    # i18n-tasks-use t('mongoid.attributes.clinic.phone')
-    # i18n-tasks-use t('mongoid.attributes.clinic.fax')
-    # i18n-tasks-use t('mongoid.attributes.clinic.active')
+    # i18n-tasks-use t('activerecord.attributes.clinic.phone')
```

Comment:
`activerecord` i18n works the same way `mongoid` i18n does, though under the activerecord namespace. We can basically square this away by copying them into this new namespace. Expect to see this for several models.